layers/meta-resin-raspberrypi/recipes-bcm/bootfiles/rpi-config_git.bbappend

```diff
@@ -5,6 +5,10 @@ do_deploy_append() {
     echo "dtparam=spi=on" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
     # Disable firmware splash by default
     echo "disable_splash=1" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+    # Disable firmware warnings showing in non-debug images
+    if [ -z "${@bb.utils.contains('DISTRO_FEATURES','debug-image','1','',d)}"]; then
```

Comment:
or this could be:

if ${@bb.utils.contains('DISTRO_FEATURES','debug-image','false','true',d)}; then